Source/Core/GoogleEarthEnterpriseTerrainData.js

```diff
@@ -147,17 +147,19 @@ define([
      * @param {Number} x The X coordinate of the tile for which to create the terrain data.
      * @param {Number} y The Y coordinate of the tile for which to create the terrain data.
      * @param {Number} level The level of the tile for which to create the terrain data.
+     * @param {SerializedMapProjection} serializedMapProjection Serialized map projection.
      * @param {Number} [exaggeration=1.0] The scale used to exaggerate the terrain.
      * @returns {Promise.<TerrainMesh>|undefined} A promise for the terrain mesh, or undefined if too many
      *          asynchronous mesh creations are already in progress and the operation should
      *          be retried later.
      */
-    GoogleEarthEnterpriseTerrainData.prototype.createMesh = function(tilingScheme, x, y, level, exaggeration) {
+    GoogleEarthEnterpriseTerrainData.prototype.createMesh = function(tilingScheme, x, y, level, serializedMapProjection, exaggeration) {
```

Comment:
Is this breaking change OK?

Reply:
Should be fine, these are private and it's been updated in the `TerrainData` interface too.